edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/AuthAgentEventIds.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
+{
+    public class AuthAgentEventIds
+    {
+        const int EventIdStart = 7000;
```

Comment:
There's still no explanation for this. Why is it 7000?

Reply:
Listed all 'EventIdStart =' in the code and picked a range not being used. It seems like a convention I copied.

Reply:
yup, thats the idea..